config/default.schema.json

```diff
@@ -57,6 +57,11 @@
 			"default": true,
 			"description": "If enabled, morgan logs are written."
 		},
+		"FEATURE_SKIP_FIRST_LOGIN_ENABLED": {
```

Comment:
This change does not look like it is part of the translation issues.
Was this part of another ticket?